app/config/questions.py

```
@@ -0,0 +1,208 @@
+"""
+Preguntas del wizard de postulación de Ithaka
+"""
+
+from typing import Dict, List, Any
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)